<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QWidget" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="insulinUnit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PowerBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="TimeDisplay">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateDisplay">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="insulinLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Insulin On Board</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BatterySpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="InsulinOnBoard">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="glucoseUnit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mmol/L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OptionsBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>291</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>OPTIONS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bolusbtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>311</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>BOLUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="BatteryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Battery</string>
   </property>
  </widget>
  <widget class="QWidget" name="GraphWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="LogGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>421</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>CGM Readings Log (5-minute intervals)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="DataListWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="PauseResumeBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SimSpeedBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulation Speed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TimeRangeBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Range: 2h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearDataBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="SimTimeLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sim Time: 00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="SimStatusLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: Running</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogTitleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>611</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>* Log shows glucose readings at 5-minute intervals (1 real second = 5 simulated minutes)</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="GlucoseLevel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ChargeBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Charge</string>
   </property>
  </widget>
  <widget class="QLabel" name="LowPowerLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Power</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
